docs/distributing-your-plugin.md

```diff
@@ -50,7 +50,7 @@ These screenshots are stored [in git: CustomAlerts/screenshots](https://github.c
 <a name="plugin-json-required-fields"></a>
 The `plugin.json` file must contain the following information:
 
-- `name`: The plugin's name. It can only contain letters (a-z), numbers (0-9) and must start with a capital letter. The name cannot contain the words `"Piwik"`, `"Core"` or `"Analytics"`. A maximum of 40 characters are allowed.
+- `name`: The plugin's name. It can only contain letters (a-z), numbers (0-9) and must start with a capital letter. The name cannot contain the words `"Piwik"`, `"Core"` or `"Analytics"`. A maximum of 60 characters are allowed.
```

Comment:
```suggestion
- `name`: The plugin's name. It can only contain letters (a-z), numbers (0-9) and must start with a capital letter. The name cannot contain the words `"Piwik"`, `"Core"` or `"Analytics"`. Since Matomo 5 a maximum of 60 characters are allowed.
```